resources/item_from_string_bc_map.json

```diff
@@ -1328,6 +1328,7 @@
     "potato_block": "minecraft:potatoes",
     "potatoes": "minecraft:potatoes",
     "potion": "minecraft:potion",
+    "powder_snow": "minecraft:powder_snow",
```

Comment:
This should not be changed - this file is kept only for BC purposes to support `LegacyStringToItemParser`